common/src/services/stateMigration.service.ts

```diff
@@ -177,9 +177,7 @@ export class StateMigrationService {
                 v1Keys.enableBiometric,
                 options
               ),
-              environmentUrls:
-                (await this.storageService.get<EnvironmentUrls>(v1Keys.environmentUrls, options)) ??
-                new EnvironmentUrls(),
+              environmentUrls: null,
```

Comment:
I'm not sure we want to remove the environment urls from the account level during these migrations. They might be used by desktop, web, and browser from here after the initial migration.

The global environmentUrls were created when we discovered that the directory connector allows environmentUrls to be set before login. We added a step in the stateMigrationService in the directory connector to migrate the environmentUrls globally as you did below, but we failed to consider that the cli may have the same issue.

Reply:
This change is in the `globals`, not the account level. I just removed it to avoid duplication given that we set it below. But I can't remove the key entirely due to type checking.

Reply:
Ah got it! I misread where it was!